### PR TITLE
[CI:DOCS] Fix troubleshooting documentation on handling supplemental groups.

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/10166

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -495,10 +495,10 @@ $ podman unshare cat /proc/self/uid_map
 
 Reference [subuid](http://man7.org/linux/man-pages/man5/subuid.5.html) and [subgid](http://man7.org/linux/man-pages/man5/subgid.5.html) man pages for more detail.
 
-### 20) Passed-in device can't be accessed in rootless container
+### 20) Passed-in devices or files can't be accessed in rootless container
 
-As a non-root user you have group access rights to a device that you want to
-pass into a rootless container with `--device=...`.
+As a non-root user you have group access rights to a device or files that you
+want to pass into a rootless container with `--device=...` or --volume=...'
```
Comment (mheon) [inline troubleshooting.md:501]: or `--volume=...`

Comment (mheon) [reply troubleshooting.md:501]: Damn, Github formatted my reply. Missing backticks, was the point I was trying to make.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**